src/sentry/api/endpoints/organization_group_index.py

```diff
@@ -164,57 +160,6 @@ class OrganizationGroupIndexEndpoint(OrganizationEventsEndpointBase):
         },
     }
 
-    @staticmethod
-    def build_better_priority_sort_kwargs(request: Request) -> Mapping[str, PrioritySortWeights]:
```

Comment:
Any chance we want to iterate more and would need this?

Reply:
I don't think so, I checked w/ Gilbert on this too.